apps/openchallenges/challenge-service/src/main/resources/db/submission_types.csv

```diff
@@ -533,3 +533,10 @@
 "532","container_image","521","2024-06-03 16:59:54"
 "533","other","521","2024-06-03 16:59:54"
 "534","prediction_file","522","2024-07-02 22:45:34"
+"535","prediction_file","523","2024-11-19 22:20:29"
+"536","notebook","523","2024-11-19 22:20:29"
+"537","prediction_file","524","2024-11-19 22:20:36"
+"538","notebook","524","2024-11-19 22:20:36"
+"539","other","524","2024-11-19 22:20:36"
+"540","prediction_file","525","2024-11-19 22:20:44"
+"541","prediction_file","526",""
```

Comment:
@mdsage1 Missing timestamp

Reply:
@tschaffter I think I've updated the spreadsheet to rectify this issue. Please let me know if it persists.